src/config/2020.json

```diff
@@ -847,16 +847,6 @@
       "github": "MaedahBatool",
       "twitter": "maedahbatool"
     },
-    "mandymichael": {
-      "name": "Mandy Michael",
```

Comment:
Mandy did contribute to the Fonts chapter, the metadata may not be up to date

Reply:
Fixed!